R/mod_load_data.R

```diff
@@ -38,6 +38,12 @@ mod_load_data_server <- function(input, output, session, cons){
       summarise_trips(., data_r$trips)
     message("Finished loading participants")
     
+    if (getOption("emdash.anon_locations"))  {
+      message("About to anonymize trajectories")
+      data_r$trips <- data_r$trips %>% anonymize_uuid
```

Comment:
We do want the tables to have UUIDs. Just not the geospatial data.

Reply:
Got it. ;)